docs/assets/cache_lifecycles.puml

```diff
@@ -61,15 +64,22 @@ Session -> "IK Cache" : cache IK
 Session -> API : complete operation 1
 |||
 API -> Session : close session
-Session -> "IK Cache" : securely close cache
-destroy "IK Cache"
+Session -> "Session Cache" : cache session
```

Comment:
this looks like when we `close session`, we cache the session whereas we decrease count of active sessions when we close a session.